oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/record-requests/components/RecordRequests.jsx

```diff
@@ -50,7 +53,8 @@ const RecordRequests = ({
   actionExtraContext,
 }) => {
   const queryClient = useQueryClient();
-  const [actionsLocked, setActionsLocked] = React.useState(false);
+  const [visibleAccordions, setVisibleAccordions] = useState([0, 1]);
```

Comment:
This state shouldn't be needed, just render the accordions based on avaliable request data